app/features/redux/middleware.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import { applyMiddleware } from 'redux';
+
+import { middleware as routerMiddleware } from '../router';
+
+import { createLogger } from 'redux-logger';
```

Comment:
ditto

Reply:
I was actually keeping this as like this.
First, `applyMiddleware()` import is from redux. (single line)
Then middleware from different features. (without line gap)
Then middleware from different packages like `redux-logger` right now. (possibly `redux-thunk` etc. later)

WDYT?

Reply:
Let's be consistent with the other files please!

Reply:
Okay. 
Then this will be fine then.
```
import { applyMiddleware } from 'redux';
import { createLogger } from 'redux-logger';

import { middleware as routerMiddleware } from '../router';
```